tools.go

```diff
@@ -0,0 +1,7 @@
+//go:build tools
```

Comment:
The docs recommend ignore here: https://arc.net/l/quote/zyjlatuf

```suggestion
//go:build ignore
```

Reply:
This causes 'go mod tidy' to clean this up https://github.com/databricks/cli/pull/2724/commits/203cc9a46fa40480ccf2787a65fd2c21c32e3727